#! /bin/sh /usr/share/dpatch/dpatch-run

## DP: Fixes pcfiles to point to CLI Policy 0.7 install locations

@DPATCH@
diff -urNad gtk-sharp2-2.12.9~/glade/glade-sharp-2.0.pc.in gtk-sharp2-2.12.9/glade/glade-sharp-2.0.pc.in
--- gtk-sharp2-2.12.9~/glade/glade-sharp-2.0.pc.in	2009-11-27 11:31:33.000000000 +0000
+++ gtk-sharp2-2.12.9/glade/glade-sharp-2.0.pc.in	2009-11-27 11:37:57.000000000 +0000
@@ -9,4 +9,4 @@
 Version: @VERSION@
 Requires: gtk-sharp-2.0
 Cflags: -I:${gapidir}/glade-api.xml
-Libs: -r:${libdir}/mono/@PACKAGE_VERSION@/glade-sharp.dll
+Libs: -r:${libdir}/cli/glade-sharp-2.0/glade-sharp.dll
diff -urNad gtk-sharp2-2.12.9~/glib/glib-sharp-2.0.pc.in gtk-sharp2-2.12.9/glib/glib-sharp-2.0.pc.in
--- gtk-sharp2-2.12.9~/glib/glib-sharp-2.0.pc.in	2009-11-27 11:31:33.000000000 +0000
+++ gtk-sharp2-2.12.9/glib/glib-sharp-2.0.pc.in	2009-11-27 11:36:31.000000000 +0000
@@ -8,5 +8,5 @@
 Description: GLib# - .NET Binding for the glib library.
 Version: @VERSION@
 Cflags: -I:${gapidir}/glib-api.xml
-Libs: -r:${libdir}/mono/@PACKAGE_VERSION@/glib-sharp.dll
+Libs: -r:${libdir}/cli/glib-sharp-2.0/glib-sharp.dll
 
diff -urNad gtk-sharp2-2.12.9~/gtk/gtk-sharp-2.0.pc.in gtk-sharp2-2.12.9/gtk/gtk-sharp-2.0.pc.in
--- gtk-sharp2-2.12.9~/gtk/gtk-sharp-2.0.pc.in	2009-11-27 11:31:33.000000000 +0000
+++ gtk-sharp2-2.12.9/gtk/gtk-sharp-2.0.pc.in	2009-11-27 11:37:40.000000000 +0000
@@ -8,5 +8,5 @@
 Description: Gtk# - GNOME .NET Binding
 Version: @VERSION@
 Cflags: -I:${gapidir}/pango-api.xml -I:${gapidir}/atk-api.xml -I:${gapidir}/gdk-api.xml -I:${gapidir}/gtk-api.xml
-Libs: -r:${libdir}/mono/@PACKAGE_VERSION@/pango-sharp.dll -r:${libdir}/mono/@PACKAGE_VERSION@/atk-sharp.dll -r:${libdir}/mono/@PACKAGE_VERSION@/gdk-sharp.dll -r:${libdir}/mono/@PACKAGE_VERSION@/gtk-sharp.dll
+Libs: -r:${libdir}/cli/pango-sharp-2.0/pango-sharp.dll -r:${libdir}/cli/atk-sharp-2.0/atk-sharp.dll -r:${libdir}/cli/gdk-sharp-2.0/gdk-sharp.dll -r:${libdir}/cli/gtk-sharp-2.0/gtk-sharp.dll
 Requires: glib-sharp-2.0
diff -urNad gtk-sharp2-2.12.9~/gtkdotnet/gtk-dotnet-2.0.pc.in gtk-sharp2-2.12.9/gtkdotnet/gtk-dotnet-2.0.pc.in
--- gtk-sharp2-2.12.9~/gtkdotnet/gtk-dotnet-2.0.pc.in	2009-11-27 11:31:33.000000000 +0000
+++ gtk-sharp2-2.12.9/gtkdotnet/gtk-dotnet-2.0.pc.in	2009-11-27 11:36:44.000000000 +0000
@@ -7,4 +7,4 @@
 Description: .NET Extensions for Gtk
 Version: @VERSION@
 Requires:gtk-sharp-2.0
-Libs: -r:${libdir}/mono/@PACKAGE_VERSION@/gtk-dotnet.dll
+Libs: -r:${libdir}/cli/gtk-dotnet-2.0/gtk-dotnet.dll
